src/pages/homepage/Homepage.js

```diff
@@ -0,0 +1,147 @@
+      productUrl: "/product/6",
+    },
+    {
+      imageUrl:
+        "https://images.samsung.com/is/image/samsung/assets/in/smartphones/galaxy-s20/galaxy-s20-fe/Homepage_banner1440.jpg",
+      productUrl: "/product/12",
+    },
+  ];
+
+  return (
+    <div className="Homepage">
+      <ImageSlider images={bannerImages} parent="homepage" />
+      <h2 className="Homepage__subHeading">Featured Products</h2>
+      <div className="Homepage__featuredProductsWrapper">
+        <div className="Homepage__featuredProductContainer">
+          <img
+            className="Homepage__featuredProdImage"
+            src="https://mir-s3-cdn-cf.behance.net/project_modules/1400_opt_1/a71331107041495.5fac2c616f3ae.jpg"
+            alt="oneplus-phone"
+          />
+          <div className="Homepage__infoContainer">
+            <div className="Homepage__featuredProdName">
+              <SiOneplus color="red" className="Homepage__featuredProdLogo" />
+              OnePlus 8T
+            </div>
+            <BuyNowTextLinks link="/product/6" />
+          </div>
+        </div>
+
+        <div className="Homepage__featuredProductContainer">
+          <div className="Homepage__infoContainer">
+            <div className="Homepage__featuredProdName">
+              <AiFillApple
+                color="black"
+                className="Homepage__featuredProdLogo"
+              />
+              iPhone 11
+            </div>
+            <BuyNowTextLinks link="/product/1" />
+          </div>
+          <img
+            className="Homepage__featuredProdImage"
+            src="https://mir-s3-cdn-cf.behance.net/project_modules/disp/230a8486312713.5d9580970f876.jpg"
+            alt="iphone11"
+          />
+        </div>
+
+        <div className="Homepage__featuredProductContainer">
+          <img
+            className="Homepage__featuredProdImage"
+            src="https://mir-s3-cdn-cf.behance.net/project_modules/1400/131e2099947973.5efde4c45921a.jpg"
+            alt="oneplus-phone"
+          />
+          <div className="Homepage__infoContainer">
+            <div className="Homepage__featuredProdName">
+              <SiHp color="#0091CF" className="Homepage__featuredProdLogo" />
+              HP Spectre x360
+            </div>
+            <BuyNowTextLinks link="/product/36" />
+          </div>
+        </div>
+      </div>
+
+      <div className="Homepage__featuredProductContainer">
+        <div className="Homepage__infoContainer">
+          <div className="Homepage__featuredProdName">
+            <AiFillApple color="black" className="Homepage__featuredProdLogo" />
+            Apple MacBook Pro
+          </div>
+          <BuyNowTextLinks link="/product/37" />
+        </div>
+        <img
+          className="Homepage__featuredProdImage macbookpro-img"
+          src="https://mir-s3-cdn-cf.behance.net/project_modules/1400_opt_1/0a175488400949.5dd52df927373.jpg"
+          alt="macbookpro"
+        />
+      </div>
```

Comment:
this can be also rendered using array store the data in array and render that, for the alternative image position you can use `flex-direction: row` and `row-reverse` on alternative index